code/game/gamemodes/objectives_subtypes.dm

```diff
@@ -0,0 +1,68 @@
+/datum/objective/debrain/science
+	name = "Debrain science"
+	target_department = DEPARTMENT_SCIENCE
+
+/datum/objective/assassinate/mindshielded
+	name = "Assassinate mindshielded"
+	flags_target = MINDSHIELDED_TARGET
+
+/datum/objective/assassinate/nomindshield
+	name = "Assassinate non-mindshielded"
+	flags_target = UNMINDSHIELDED_TARGET
+
+/datum/objective/assassinate/syndicate
+	name = "Assassinate syndicate agent"
+	flags_target = SYNDICATE_TARGET
+
+/datum/objective/assassinate/syndicate/update_explanation_text()
+	..()
+	if(target?.current)
+		explanation_text = "Assassinate [target.current.real_name], the Syndicate agent undercover as the [target.assigned_role]."
+
+/datum/objective/assassinate/medical
+	name = "Assassinate medical"
+	target_department = DEPARTMENT_MEDICAL
+
+/datum/objective/assassinate/engineering
+	name = "Assassinate engineering"
+	target_department = DEPARTMENT_ENGINEERING
+
+/datum/objective/assassinateonce/animal_user
+	name = "Assassinate animal user"
```

Comment:
... assasinate animal user? Do they do lines of cokepybaras?

Reply:
Maybe animal product user?

Reply:
(I think they meant *ab*user)

Reply:
This is admin tools/code viewable only but sure ill change it to that